sycl/test-e2e/Graph/vendor_test_macro.cpp

```diff
@@ -0,0 +1,14 @@
+// RUN: %{build} -o %t.out
```

Comment:
Should be moved to a regular `sycl/test` folder: we don't really need any devices for this test and we can convert it to compile-only test using `static_assert` instead of `assert`

Reply:
I've moved this test to the `sycl/test/extensions` folder, and modified it to be in-line with the other macro tests there.